engine/src/deploy.cpp

```diff
@@ -127,6 +127,12 @@ static Exec_stat MCDeployPushMinOSVersion(MCDeployParameters& p_params, MCDeploy
     t_major = t_minor = t_inc = 0;
     sscanf(p_vers_string, "%d.%d.%d", &t_major, &t_minor, &t_inc);
     
+    
+    // SN-2015-02-12L: [[ Bug 14422 ]] We do not push anything if the minimum
+    //  version less than 7.0
+    if (t_major < 7)
```

Comment:
I don't think this is correct - if the min_os key is not present in the deploy info array then it should have no effect on the binary. Thus, this would be better controlled in the standalone builder script.